src/main/java/frc/robot/RobotContainer.java

```diff
@@ -277,15 +286,13 @@ private void configureBindings() {
                 CommandUtil.logged(new StopIntake(intake)),
                 CommandUtil.logged(new StopFeeder(feeder))));
 
-        controller.rightTrigger().onTrue(new ParallelDeadlineGroup(
-                new SequentialCommandGroup(
-                        new WaitCommand(0.5),
-                        new FeederGamepieceUntilLeave(feeder,ramp),
-                        new WaitCommand(GameConstants.SHOOTER_TIME_BEFORE_STOPPING),
-                        new RampMove(ramp, () -> GameConstants.RAMP_POS_STOW)
-                ),
-                new AdvancedSpinningShot(shooter,() -> drivetrain.getPose(), ()-> drivetrain.getAlignable())
-        ));
+        SequentialCommandGroup advancedShoot = new SequentialCommandGroup(
+                new FeederGamepieceUntilLeave(feeder, ramp),
+                new WaitCommand(GameConstants.SHOOTER_TIME_BEFORE_STOPPING),
+                new RampMove(ramp, () -> GameConstants.RAMP_POS_STOW)
+        );
+        controller.povRight().onTrue(advancedShoot);
```

Comment:
I don't think we should use the same command instance in two bindings - there can be state changes that are unpredictable from the command being scheduled from two different places.

Reply:
removed from operator controller